nemo/collections/asr/models/sortformer_diar_models.py

```diff
@@ -557,13 +558,13 @@ def test_batch(
                     audio_signal=audio_signal,
                     audio_signal_length=audio_signal_length,
                 )
+                self._get_aux_test_batch_evaluations(batch_idx, preds, targets, target_lens)
```

Comment:
It is a bit difficult to understand when the lists (e.g., `self.preds_total_list`, `self.batch_f1_accs_list`) are initialized in this function but are updated inside the `self. _get_aux_test_batch_evaluations`, but not a strong opinion~